components/scripts/lib/config.sh

```diff
@@ -116,6 +120,7 @@ collect_git_details() {
   collect_git_repo
   collect_git_branch
   collect_git_commit_id
+  collect_git_options
```

Comment:
I had not thought about asking for this in interactive mode. 🤔 . I could go either way with this here. On the one hand, we ask for additional arguments to pass to Gradle/Maven, and this is similar. On the other hand, it should be uncommon for a user to set `--git-options`, so I don't know how important it is to ask for it in interactive mode.

I'll be interested in what Etienne thinks.

Reply:
Nevermind, I see that Etienne asked for this already. 👍